build_tools/waf_dynamo.py

```diff
@@ -508,7 +508,12 @@ def default_flags(self):
         emflags_link =[j for i in emflags_link for j in i]
 
         flags = []
+        if os.environ.get('EMCFLAGS', None) is not None:
+            flags += os.environ.get("EMCFLAGS", "").split(' ')
         linkflags = []
+        if os.environ.get('EMLINKFLAGS', None) is not None:
+            linkflags += os.environ.get("EMLINKFLAGS", "").split(' ')
```

Comment:
Would it make more sense to use `CFLAGS` and `LDFLAGS` to make it work for other platforms?